tests/test_cli/test_analytics.py

```diff
@@ -110,3 +110,18 @@ def test_opt_in_subcommand(
     # Check the contents of the config file match
     with open(config_file_path) as f:
         assert dict(yaml.safe_load(f)) == expected_configuration
+
+
+def test_cli__analytics_command_no_args(runner):
+    """Test cli analytics command when no argument passed."""
```

Comment:
Docstring should include `Args` and the summary for the function could be more description: "Test CLI analytics command displays the help message when no arguments are passed."